extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -191,6 +192,12 @@ describe('amp-a4a', () => {
     expect(child.src).to.match(/^https?:[^?#]+nameframe(\.max)?\.html/);
     const nameData = child.getAttribute('name');
     expect(JSON.parse.bind(null, nameData), nameData).not.to.throw(Error);
+    const attributes = JSON.parse(nameData);
+    expect(attributes).to.be.ok;
+    expect(attributes._context).to.be.ok;
+    if (!attributes._context.amp3pSentinel) {
+      expect(attributes._context.sentinel).to.be.ok;
+    }
```

Comment:
So here you're saying that at least one of these should be present, right?  Do you want to ensure that _exactly one_ of them is present?

Perhaps a better way to structure this is to add an argument to `verifyNameFrameRender` that specifies which you should expect, given the case being tested.  Then you can do something like:
```javascript
function verifyNameFrameRender(element, expectSentinel) {
  // stuff
  expect(attributes._context).not.to.contain.all.keys('sentinel', 'amp3pSentinel');
  if (expectSentinel) {
    expect(attributes._context).to.contain.keys('sentinel');
    expect(attributes._context.sentinel).to.match(/some_regex/);
  } else {
    expect(attributes._context).to.contain.keys('amp3pSentinel');
    expect(attributes._context.amp3pSentinel).to.match(/some_regex/);
  }
```